### PR TITLE
Refactor vue

### PR DESCRIPTION
**Pull request by mf222**

### CODE REVIEW

File: app/javascript/packs/application.js [hunk 1]
```diff
@@ -1,20 +1,25 @@
...
-  const app = new Vue({
-    render: h => h(App),
+
+  let renderElement = null;
+  let idElement = '';
+
+  Vue.component('main-app', {});
+  Vue.component('selector', Selector);
+
+  new Vue({
```
Comment (monkeyci) [inline app/javascript/packs/application.js]: Do not use 'new' for side effects                         no-new

File: app/javascript/packs/application.js [hunk 2]
```diff
@@ -1,20 +1,25 @@
...
 
 Vue.use(VueI18n);
 
 document.addEventListener('DOMContentLoaded', () => {
-  document.getElementById('selector').appendChild(document.createElement('selector'));
-  const app = new Vue({
-    render: h => h(App),
+
+  let renderElement = null;
+  let idElement = '';
```
Comment (monkeyci) [inline app/javascript/packs/application.js]: 'idElement' is assigned a value but never used            no-unused-vars<br>'idElement' is never reassigned. Use 'const' instead      prefer-const

File: app/javascript/packs/application.js [hunk 3]
```diff
@@ -1,20 +1,25 @@
...
 import Locales from '../locales.js';
 
 Vue.use(VueI18n);
 
 document.addEventListener('DOMContentLoaded', () => {
-  document.getElementById('selector').appendChild(document.createElement('selector'));
-  const app = new Vue({
-    render: h => h(App),
+
+  let renderElement = null;
```
Comment (monkeyci) [inline app/javascript/packs/application.js]: 'renderElement' is assigned a value but never used        no-unused-vars<br>'renderElement' is never reassigned. Use 'const' instead  prefer-const

File: app/javascript/packs/application.js [hunk 4]
```diff
@@ -1,20 +1,22 @@
...
 
 document.addEventListener('DOMContentLoaded', () => {
-  document.getElementById('selector').appendChild(document.createElement('selector'));
-  const app = new Vue({
-    render: h => h(App),
+
+  Vue.component('main-app', {});
+  Vue.component('selector', Selector);
+
+  new Vue({ //eslint-disable no-new
```
Comment (monkeyci) [inline app/javascript/packs/application.js]: Do not use 'new' for side effects            no-new<br>Expected space or tab after '//' in comment  spaced-comment

File: app/views/organizations/show.html.erb [hunk 5]
```diff
@@ -1,69 +1,73 @@
...
+      <div class="info">
         <p class="info__description">
-            <%= t('messages.dashboard.greeting') %> <b><%=  @auth_login %></b>(<%= link_to "logout", "/oauth"%>)! <%=  t('messages.dashboard.info')%>
+          <%= t('messages.dashboard.greeting') %> <b><%=  @auth_login %></b>(<%= link_to "logout", "/oauth"%>)! <%=  t('messages.dashboard.info')%>
         </p>
-    </div>
-    <div class="organization__selector">
-        <div id="selector"></div>
+      </div>
+      <div class="organization__selector" data-options="hola">
```
Comment (gmq) [inline app/views/organizations/show.html.erb]: Se te pasó este `data-options`

Comment (mf222) [reply app/views/organizations/show.html.erb]: Gracias por notarlo!! 😱

**Comment by poxvirus** [comment]
[Cambiar de organización](https://trello.com/c/k1a87RIq/64-cambiar-de-organizaci%C3%B3n)

**Pull request merged**